### PR TITLE
Small updates to AreaSelector and Dropdown

### PR DESCRIPTION
**Pull request by joningehe**
- Dropdown takes in more optional params to allow more customization
- AreaSelector updated to new design

### CODE REVIEW

File: src/components/UI/dropdown/dropdown.module.scss [hunk 1]
```diff
@@ -32,3 +32,43 @@
...
+        }
+
+        img {
+            width: 1.5rem;
+            height: 1.5rem;
+        }
+    }
+}
+
+@media only screen and (max-width: 884px) {
```
Comment (trulshj) [inline src/components/UI/dropdown/dropdown.module.scss:59]: Breakpoint kunne vært globalt variabel

File: src/components/UI/dropdown/index.tsx [hunk 2]
```diff
@@ -13,9 +13,11 @@ type DropdownProps = {
         label: string
     }[]
     disabled?: boolean
+    borderBottom?: boolean
+    size?: 'default' | 'large'
 }
 
-export default function Dropdown({
+export default function Drodown({
```
Comment (trulshj) [inline src/components/UI/dropdown/index.tsx:20]: Skriveleif?

**Review by trulshj — APPROVED** [review]
To småting, ellers bra! 

lachlan-murdoch-reaction-gif-by-murdoch-mysteries

File: src/components/UI/dropdown/index.tsx (hunk 2 above)
Comment (joningehe) [reply src/components/UI/dropdown/index.tsx]: Fikset

**Pull request merged**